### DISCUSSION TITLE
Online books

### DISCUSSION THREAD

**Discussion by ali-ghamdan** [PaulWoitaschek/Voice#2527] [Ideas]
Support online books
Like YouTube playlists or videos
Some playlists are for reading books
So it will be very good and helpful
Without download any content
Will be very helpful

**Comment by SierraGuru** [comment]
Do you want to stream from a personal server like Jellyfin, a cloud source like Google Drive, or just keep a list of bookmarks?

**Reply by ali-ghamdan** [reply]
the list saved in my phone but the stream is recived from youtube
